mbuild/formats/lammpsdata.py

```diff
@@ -100,17 +102,19 @@ def write_lammpsdata(structure, filename):
     with open(filename, 'w') as data:
         data.write(filename+' - created by mBuild\n\n')
         data.write('{:d} atoms\n'.format(len(structure.atoms)))
-        data.write('{:d} bonds\n'.format(len(bonds)))
-        data.write('{:d} angles\n'.format(len(angles)))
-        data.write('{:d} dihedrals\n\n'.format(len(dihedrals)))
+        if atom_style == 'full' or atom_style == 'molecular':
```

Comment:
Is `if atom_style in ['full', 'molecular']` a better way to do this? A few similar cases below as well.